sysutils/pkg/Portfile

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        freebsd pkg 1.10.4
+github.tarball_from releases
+categories          sysutils archivers
+platforms           darwin
+license             BSD
+maintainers         Email: ato.ms:apowers
+description         Package management tool for FreeBSD
+long_description    Pkg is the Next Generation package management tool for FreeBSD. Its main goals are to faciliate remote binary package upgrades. It also works with ports without remote binary packages.
+homepage            https://wiki.freebsd.org/pkgng
+
+depends_lib         port:libarchive
+
+checksums           sha256 5669c3727e980885922402232268f1023bfc13b02bb3dab966f14e99266ad3eb
```

Comment:
We require at least 2 checksums. Add an `rmd160` checksum and the `size` parameter.

Reply:
Added rmd160 in latest commit; can't find any example of the `size` parameter.

Reply:
The size of the file, in bytes.